scripts/get-arb-relay-cost.sh

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env bash
+set -e
+
+[[ "$(cast chain --rpc-url="$ETH_RPC_URL")" == "goerli" ]] || { echo "Please set a Goerli ETH_RPC_URL"; exit 1; }
+
+ARBITRUM_GOERLI_RPC_URL='https://goerli-rollup.arbitrum.io/rpc'
+
+L2_SPELL='0x11dc6ed4c08da38b36709a6c8dbaac0eaedd48ca'
+CHANGELOG='0xdA0Ab1e0017DEbCd72Be8599041a2aa3bA7e740F'
+NODE_INTERFACE='0x00000000000000000000000000000000000000C8'
+
+L1_GOV_RELAY=$(
+    cast call $CHANGELOG "getAddress(bytes32)(address)" \
+    $(cast --to-bytes32 $(cast --from-ascii "ARBITRUM_GOV_RELAY"))
```

Comment:
This can be simplified with: 
```suggestion
    $(cast --format-bytes32-string "ARBITRUM_GOV_RELAY"))
```
(i think i got the formatting correct this time

Reply:
did you?  I think you needed to delete that second closing paren `)`.

Reply:
i missed the one on the new line, should be fixed now